spec/features/audit/allocation/allocate_spec.rb

```diff
@@ -38,7 +38,7 @@
   scenario 'Allocate all content within current page', :js do
     create_list(:content_item, 26)
 
-    visit audits_allocations_path
+    visit audits_allocations_path(allocated_to: "anyone")
```

Comment:
I think these scenarios should start with the default state which is to visit the allocations page without any filters. We should then describe what happens initially, then how the behaviour changes when we filter.